lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -219,6 +219,20 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
         return this._tagNameControl;
     }
 
+    /*
+     * Returns `true` if tagNameControlVisible is true, otherwise `false`
+    */
+    get isNameTagsVisible(): boolean {
+        return (!this.tagNameControlVisible && this.tags?.length > 0) || this.tagNameControlVisible;
+    }
+
+    /*
+     * Returns `true` if tags empty, otherwise `false`
+    */
+    get isTagsEmpty(): boolean {
+        return this.tags.length === 0 && !this.tagNameControlVisible;
```

Comment:
add the null check here.
return this.tags?.length === 0

Reply:
done